src/components/ShowList.jsx

```diff
@@ -22,6 +21,30 @@ function ShowList() {
     setFilter(e.target.value);
   };
 
+  const inactiveItem = (item) => {
```

Comment:
A common pattern for functions that return a boolean is to name them with an `is` prefix, so this case would look like: `isInactiveItem`